ui/src/apps/dashboardSettings/RootComponent.js

```diff
@@ -1,5 +1,7 @@
 import React from 'react'
-import { Form, Select } from 'antd'
+import { Form } from '@ant-design/compatible';
+import '@ant-design/compatible/assets/index.css';
```

Comment:
Not sure whether using css will make the modified theme color in config-overrides.js doesn't work anymore, need to confirm it.

Reply:
You can change the usage of `Form` according to antd's guideline directly so that there will be no need to introduce the compatible part. Dashboard does not heavily rely on the form functionality and it should be very easy to migrate manually.

Reply:
got it.